ostd/src/bus/mmio/common_device.rs

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! MMIO device common definitions or functions.
+
+use int_to_c_enum::TryFromInt;
+use log::info;
+
+use super::VIRTIO_MMIO_MAGIC;
+use crate::{
+    io_mem::IoMem,
+    mm::{paddr_to_vaddr, Paddr, VmIo},
+    trap::IrqLine,
+};
+
+/// MMIO Common device.
+/// TODO: Implement universal access to MMIO devices since we are temporarily
+/// using specific virtio device as implementation of CommonDevice.
```

Comment:
TODO are almost always written for developers, not users. So TODOs should be written as comments, not Rust docs.